index.html

```diff
@@ -4,10 +4,47 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Personal Portfolio</title>
+    <title>Document</title>
     <link rel="stylesheet" href="style.css">
 </head>
 <body>
-    
+    <section id="Home">
+        <nav>
+            <div class="logo">
+                <img src="" alt="">
+                <h2>My logo</h2>
+            </div>
+            <div class="navbar_links">
+                <a href="#portfolio">Portfolio</a>
+                <a href="#about">About</a>
+                <a href="#contact">Contact</a>
+                <a href="#"><img src="assets/Disabled.png" alt=""></a>
+            </div>
+            <div class="phone_navbar">
+                <img src="assets/Icon - Menu.png" alt="">
```

Comment:
- [ ] Please, kindly make sure that the menu/hamburger icon is contained in a `nav` tag for semantic and search engine reason 😃.